cli/integration_tests/basic_monorepo/profile.t

```diff
@@ -0,0 +1,9 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd)
+
+Run build and record a trace
+Ignore output since we want to focus on testing the generated profile
+  $ ${TURBO} build --profile=build.trace > turbo.log
+Make sure the resulting trace is valid JSON
+  $ node -e "require('./build.trace')"
```

Comment:
just making sure i read this right. if this has non-zero exit, the test fails?

Reply:
Yup, if the `node -e "require('./build.trace')"` outputs anything the test will fail.